### ISSUE TITLE
Zeitanzeige im '

### ISSUE THREAD

**Issue by Zeppelin-PV** [solectrus/solectrus#2152]

**Comment by Zeppelin-PV** [comment]
wäre es möglich und denkbar dass die Zeitanzeige im 'Jetzt' (die ja immer die aktuelle Stunde abbildet) bei gedrückter Maustaste
nach links und rechts verschiebbar zu machen ? (maximal bis zur Tagesgrenze)
Das hätte einen großen Vorteill wenn man mal kurz zwei Stunden zurückschauen möchte und die Anzeige im 'Tag' zu grob dafür ist.
Für den Desktop sicherlich sinnvoll.

**Comment by ledermann** [comment]
Ja, das ist denkbar, dürfte aber etwas knifflig zu realisieren sein. Ich lasse es erst mal so stehen.

**Comment by MartinKBroll** [comment]
> wäre es möglich und denkbar dass die Zeitanzeige im 'Jetzt' (die ja immer die aktuelle Stunde abbildet) bei gedrückter Maustaste nach links und rechts verschiebbar zu machen ? (maximal bis zur Tagesgrenze) Das hätte einen großen Vorteill wenn man mal kurz zwei Stunden zurückschauen möchte und die Anzeige im 'Tag' zu grob dafür ist. Für den Desktop sicherlich sinnvoll.

Aber gerade auf dem Desktop ist es doch mit der Maus problemlos möglich, einen beliebigen Zeitabschnitt größer zu ziehen?

**Comment by ledermann** [comment]
@MartinKBroll Das Knifflige beim oben beschrieben "Schieben" ist, dass ja Daten nachgeladen werden müssen. Das ist natürlich machbar, aber eben nicht so einfach.

Übrigens: Das derzeitige Zoomen sorgt nicht für eine höhere Auflösung. Es wird nur das vorhandene Diagramm hochskaliert, es erscheinen keine zusätzlichen Datenpunkte.
